### ISSUE TITLE
Changed indent after `@section` with multiple arguments (from version 1.11.13)

### ISSUE THREAD

**Issue by kawahara-kunio** [shufo/blade-formatter#326]
Changed indent after `@section` with multiple arguments (from version 1.11.13)
Is this intended behavior?

## Expected Behavior
Before 1.11.13
```blade
@section('content', 'title')
  <div class="some-class">
  </div>
@endsection
```

## Current Behavior
From 1.11.13
```blade
@section('content', 'title')
<div class="some-class">
</div>
@endsection
```

**Comment by schelmo** [comment]
but isnt your code invalid? 
if the second parameter is filled, the section is already ended and you will get an exception:
```
InvalidArgumentException
Cannot end a section without first starting one.
```

**Comment by shufo** [comment]
I also encountered an exception on Laravel v8.54.0

```bash
❯ composer show | grep laravel/framework
laravel/framework                  v8.54.0 The Laravel Framework.
```

```blade
<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <title>Laravel</title>

    <!-- Fonts -->
    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@400;600;700&display=swap" rel="stylesheet">
~~~~
</body>
@section('content', 'title')
<div class="some-class">
</div>
@endsection

</html>
```

then

![2021-08-15_23-55](https://user-images.githubusercontent.com/1641039/129482681-89b89154-c929-4832-8049-7257e63778b7.png)

Laravel 6.x also occured exception on above template.

![2021-08-15_23-58](https://user-images.githubusercontent.com/1641039/129482809-e13688db-6035-4305-9664-b08b12e4e30d.png)

**Comment by kawahara-kunio** [comment]
Sorry, example above is not good.
Actually, it is a more complicated structure as shown below.
A blade file uses `extends` with `layouts/test.blade.php`


somePage.blade.php
```blade
@extends('layouts.test')

@section('title', 'This is title')

@section('content')
  <div class="someClass">
    This is content.
  </div>
@endsection
```

layouts/test.blade.php
```blade
<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>@yield('title')</title>
</head>

<body>
  <div class='content'>
    @yield('content')
  </div>
</body>

</html>
```

With this situation, from 1.11.13, the blade file (somePage.blade.php) will be formatted as shown below.
```blade
@extends('layouts.test')

@section('title', 'This is title')

@section('content')
<div class="someClass">
  This is content.
</div>
@endsection
```

**Comment by shufo** [comment]
Fixed by #331. Thanks @schelmo! 
And release is at https://github.com/shufo/blade-formatter/releases/tag/v1.12.1

Thanks for reporting the issue @kawahara-kunio!
